README.md

```diff
@@ -10,6 +10,20 @@ It is part of [Membrane Multimedia Framework](https://membraneframework.org).
 
 ## Installation
 
+First, you need to install FFmpeg on your system:
+
+### macOS
+
+```shell
+brew install ffmpeg
+```
+
+### Ubuntu
+
+```shell
+sudo apt-get install ffmpeg
+```
+
```

Comment:
I don't think FFmpeg is needed here - as far as I can see, FFmpeg H.264 decoder is only used in tests (and that is why we have `{:membrane_h264_ffmpeg_plugin, :test}` dependency in `mix.exs`. 
The `:test` dependencies are needed only for development, so I would say that we can add that section, but emphasise that the FFmpeg's installation is required only in case of development.

Reply:
You're right, because I tested this package by running `mix test` or `mix test --include manual` and there it was required. I will emphasise that is only for development purpose needed.